### ISSUE TITLE
[feature] NFS as Samba alternative

### ISSUE THREAD

**Issue by robbyb67** [openhab/openhabian#218]
I've successfully replaced all Windows installations in my home network. 
So I would prefer sharing the openhab directories by NFS rather than samba in openhabian-config.

**Comment by ThomDietrich** [comment]
Hey robby,
there is actually just one active samba share for the directory `/srv`. Go ahead and share it.
Did you create the ticket because you think a pre-configured NFS sharing in parallel to Samba would be worth it? I don't use NFS but feel free to contribute a PR! 👍

**Comment by mstormi** [comment]
To get this right (with all those options and things that may go wrong) is quite a challenge to any script writer.
Meanwhile I feel this is useful just to a small minority of people if any. Those to have eliminated Windoze sure do know how to accomplish this manually, they don't need a openHABian GUI.
Close ?

**Comment by ThomDietrich** [comment]
I'm with you. Most will not need this. However if someone wants to add the feature, be my guest. That is why I'd leave it open but added the "nice to have" label. ok?

**Comment by mstormi** [comment]
After almost 2 yrs w/o input and in the light of our recent decision to focus on reducing complexity, I think we should close this. Don't worry, the Internet (and Github) never forgets.
